core/controllers/questions_list.py

```diff
@@ -47,7 +47,12 @@ class QuestionsListHandler(base.BaseHandler):
     @acl_decorators.open_access
     def get(self, comma_separated_skill_ids):
         """Handles GET requests."""
-        start_cursor = self.request.get('cursor')
+        more = True
```

Comment:
This doesn't really have any context when placed here, please put it much lower down just before it gets used.

Reply:
Removing 'more'.